colors/wal.vim

```diff
@@ -189,6 +189,15 @@ hi ALEWarning ctermbg=NONE ctermfg=3
 
 " }}}
 
+" Neovim treesitter {{{
+
+hi TSInclude ctermbg=NONE ctermfg=5
```

Comment:
You can replace these with 
```vim
exe 'hi TSInclude guibg=NONE ctermbg=NONE ctermfg=5 guifg='.g:color5
" ...
```
So that it also supports when you do `set termguicolors`